docs/CreateEdit.md

```diff
@@ -2282,51 +2274,6 @@ const dataProvider = {
 }
 ```
 
-### Using `onSave` To Alter the Form Submission Behavior
```

Comment:
The proper way to do this must be documented somewhere

Reply:
We already have documentation for `transform` right ?

Reply:
right